Graphics/Implicit/Export/Render/HandleSquares.hs

```diff
@@ -3,6 +3,8 @@
 -- Copyright (C) 2016, Julia Longtin (julial@turinglace.com)
 -- Released under the GNU AGPLV3+, see LICENSE
 
+{-# LANGUAGE LambdaCase #-}
```

Comment:
Why is this here?

Reply:
Required for `\case` bellow. I don't like to write `\x -> case x of`. Maybe we could enable some harmless extensions globally as well but per file is fine for now, although missing `ScopedTypeVariables` sometimes lead to a lot head scratching.